### PR TITLE
feat: add script for test trigger deploys

### PR DESCRIPTION
**Pull request by chad-js**
these scripts contain the configs we use for our phased testing. this should help make things faster when we need to redeploy, instead of having to manage and copy paste into [DeployUMATrigger.s.sol](https://github.com/Cozy-Finance/cozy-v2-triggers/blob/main/script/DeployUMATrigger.s.sol) for each trigger and running separately

tested locally on anvil

if we're cool w this, I have another pr ready for the protection set we always deploy

### CODE REVIEW

**Review by mds1 — APPROVED** [review]

File: script/test/DeployTestChainlinkTriggers.s.sol [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+        )
+      );
+      console2.log(
+        "ChainlinkTrigger deployed",
+        _availableTrigger
+      );
+    } else {
+      // A trigger exactly like the one you wanted already exists!
+      // Since triggers can be re-used, there's no need to deploy a new one.
+      console2.log("Found existing trigger with specified configs.");
```
Comment (davidlaprade) [inline script/test/DeployTestChainlinkTriggers.s.sol]: Would you want to log the address in this case?

```suggestion
      console2.log("Found existing trigger with specified configs", _availableTrigger);
```

File: script/test/DeployTestUMATriggers.s.sol [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+        )
+      );
+      console2.log(
+        "UMATrigger deployed",
+        _availableTrigger
+      );
+    } else {
+      // A trigger exactly like the one you wanted already exists!
+      // Since triggers can be re-used, there's no need to deploy a new one.
+      console2.log("Found existing trigger with specified configs");
```
Comment (davidlaprade) [inline script/test/DeployTestUMATriggers.s.sol]: Same here:

```suggestion
      console2.log("Found existing trigger with specified configs", _availableTrigger);
```

**Review by davidlaprade — APPROVED** [review]
Looks great! Just one small suggestion, shouldn't block merge

File: script/test/DeployTestUMATriggers.s.sol (hunk 2 above)
Comment (chad-js) [reply script/test/DeployTestUMATriggers.s.sol]: [dd86be6](https://github.com/Cozy-Finance/cozy-v2-triggers/pull/24/commits/dd86be635b4f00a526b4abf2330933d6d8b1c2d1)

File: script/test/DeployTestChainlinkTriggers.s.sol (hunk 1 above)
Comment (chad-js) [reply script/test/DeployTestChainlinkTriggers.s.sol]: [dd86be6](https://github.com/Cozy-Finance/cozy-v2-triggers/pull/24/commits/dd86be635b4f00a526b4abf2330933d6d8b1c2d1)

**Pull request merged**